.github/workflows/ci-cd.yml

```diff
@@ -199,8 +227,18 @@ jobs:
       - name: Build Knossos UI
         run: ${{ matrix.script }} client-ui-build
 
+      - name: Compress artifacts
```

Comment:
I would ditch the archive change for now. I have a more elegant way to do this I'll send a PR for if you really want compressed archives.